src/util/triplea/MapXMLCreator/GameSequencePanel.java

```diff
@@ -0,0 +1,186 @@
+import javax.swing.SwingUtilities;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public class GameSequencePanel extends DynamicRowsPanel {
+	
+	public GameSequencePanel(final JPanel stepActionPanel)
+	{
+		super(stepActionPanel);
+	}
+	
+	public static void layout(final MapXMLCreator mapXMLCreator, final JPanel stepActionPanel)
+	{
+		if (s_me == null || !(s_me instanceof GameSequencePanel))
+			s_me = new GameSequencePanel(stepActionPanel);
+		DynamicRowsPanel.layout(mapXMLCreator, stepActionPanel);
+	}
+	
+	protected ActionListener getAutoFillAction()
+	{
+		return new ActionListener() {
+			
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				if (JOptionPane.YES_OPTION == JOptionPane.showConfirmDialog(m_ownPanel, "Are you sure you want to use the  Auto-Fill feature?\r\nIt will remove any information you have entered in this step and propose commonly used choices.", "Auto-Fill Overwrite Warning", JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE)) {
+					MapXMLHelper.s_gamePlaySequence.clear();
+					MapXMLHelper.s_gamePlaySequence.put("bid", new ArrayList<String>(Arrays.asList("BidPurchaseDelegate", "Bid Purchase")));
+					MapXMLHelper.s_gamePlaySequence.put("placeBid", new ArrayList<String>(Arrays.asList("BidPlaceDelegate", "Bid Placement")));
+					MapXMLHelper.s_gamePlaySequence.put("tech", new ArrayList<String>(Arrays.asList("TechnologyDelegate", "Research Technology")));
+					MapXMLHelper.s_gamePlaySequence.put("tech_Activation", new ArrayList<String>(Arrays.asList("TechActivationDelegate", "Activate Technology")));
+					MapXMLHelper.s_gamePlaySequence.put("purchase", new ArrayList<String>(Arrays.asList("PurchaseDelegate", "Purchase Units")));
+					MapXMLHelper.s_gamePlaySequence.put("move", new ArrayList<String>(Arrays.asList("MoveDelegate", "Combat Move")));
+					MapXMLHelper.s_gamePlaySequence.put("battle", new ArrayList<String>(Arrays.asList("BattleDelegate", "Combat")));
+					MapXMLHelper.s_gamePlaySequence.put("place", new ArrayList<String>(Arrays.asList("PlaceDelegate", "Place Units")));
+					MapXMLHelper.s_gamePlaySequence.put("endTurn", new ArrayList<String>(Arrays.asList("BidPurchaseDelegate", "Turn Complete")));
+					// Update UI
+					SwingUtilities.invokeLater(new Runnable()
+					{
+						
+						@Override
+						public void run()
+						{
+							resetRows();
+							m_ownPanel.revalidate();
+							m_ownPanel.repaint();
+							m_ownPanel.requestFocus();
+						}
+					});
+				}
+			}
+		};
+	}
+	
+	protected void layoutComponents()
+	{
+		
+		final JLabel lSequenceName = new JLabel("Sequence Name");
+		Dimension dimension = lSequenceName.getPreferredSize();
+		dimension.width = 140;
+		lSequenceName.setPreferredSize(dimension);
+		final JLabel lClassName = new JLabel("Class Name");
+		lClassName.setPreferredSize(dimension);
+		final JLabel lDisplayName = new JLabel("Display Name");
+		lDisplayName.setPreferredSize(dimension);
+		
+		// <1> Set panel layout

```

Comment:
Pls avoid the numbering, it makes updating the code more difficult.

Breaking up the methods into helper methods could replace the comments, and help to create some abstraction/hide some of the complexity.